pages/about/principal.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Mount Sion CBS Secondary School</title>
+    
+    <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js">
+   <script src="//code.jquery.com/jquery-2.2.1.js"></script>
+
+</head>
+ <body>
+    
+<div class="ui container">
+    <img class="ui centered medium image" src="../../assets/images/logo.jpg">
+        
+<div class="stackable ui tabular menu">
```

Comment:
I am thinking with the dropdown menu's the tabular menu does not look great, might be worth trying a different style menu